app/src/main/java/ch/sdp/vibester/database/UsersRepo.kt

```diff
@@ -16,23 +16,53 @@ import javax.inject.Inject
 class UsersRepo @Inject constructor() {
     private val dbRef = Database.get().getReference("users")
 
-
     /**
-     * The users class which handled all the interactions with the database that are linked to users
+     * This function updates a specific field of a user in the database
      * @param userID the id of the user which is being updated
      * @param newVal the new value of the field that is being updated
      * @param fieldName the field name of the field that is being updated
      */
     fun updateField(userID: String, newVal: String, fieldName: String) {
-        dbRef.child(userID) //For now ID is hardcoded, will generate it creating new users next week "-Myfy9TlCUTWYRxVLBsQ"
+        dbRef.child(userID) //For now ID is hardcoded, will generate it creating new users next week "testUser"
             .child(fieldName)
             .setValue(newVal)
     }
 
     /**
-     * The users class which handled all the interactions with the database that are linked to users
+     * This function creates a new user account in the database
+     * @param email the email of the new user
+     * @param username the username of the new user
+     * @param handle the handle of the new user
+     * @param callback function to be called when the the user has been created
+     */
+    fun createUser(email: String, username: String, handle: String, callback: (String) -> Unit) {
+        var newUser = UserProfile(
+            handle,
```

Comment:
black magic: if you put the arguments in one line, you can increase the coverage.
since they are all very short, this will not affect the visibility much.